.github/workflows/ci.yaml

```
@@ -57,11 +57,12 @@ jobs:
           docker build -t delphi_web_epidata -f ./devops/Dockerfile .
           cd ../../../
 
+      # MODULE_NAME is used to set the location of the main.py file, see https://github.com/tiangolo/meinheld-gunicorn-docker#module_name
```

how about this text?  making note of the variable name is important:
```suggestion
      # MODULE_NAME specifies the location of the `app` variable, the actual WSGI application object to run.
      # see https://github.com/tiangolo/meinheld-gunicorn-docker#module_name
```